app/assets/javascripts/koi/controllers/image_field_controller.js

```diff
@@ -1,23 +1,55 @@
-import { Controller } from "@hotwired/stimulus";
+import {Controller} from "@hotwired/stimulus";
 
 export default class ImageFieldController extends Controller {
-  static targets = ["preview"];
+  static targets = ["preview", "destroyImage"];
   static values = {
     mimeTypes: Array,
   };
 
   #counter = 0;
+  #initialImageSrc;
+  #onUploadFlag = false;
```

Comment:
what is this sorcery?

Reply:
These are private class variables in JS 😄 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields